src/Sql/dbo/Vault/Stored Procedures/SecurityTask/SecurityTask_ReadByOrganizationIdMetrics.sql

```diff
@@ -0,0 +1,17 @@
+CREATE PROCEDURE [dbo].[SecurityTask_ReadByOrganizationIdMetrics]
+    @OrganizationId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
+        COUNT(CASE WHEN ST.[Status] = 1 THEN 1 END) AS CompletedTasksCount,
+        COUNT(*) AS TotalTasksCount
+    FROM
+        [dbo].[SecurityTaskView] ST
```

Comment:
```suggestion
        [dbo].[SecurityTaskView] st
```
⛏️ We lowercase these.